dapp/src/wagmiConfig.ts

```diff
@@ -24,12 +28,32 @@ const transports = chains.reduce(
 const orangeKitUnisatConnector = getOrangeKitUnisatConnector(connectorConfig)
 const orangeKitOKXConnector = getOrangeKitOKXConnector(connectorConfig)
 const orangeKitXverseConnector = getOrangeKitXverseConnector(connectorConfig)
+const orangeKitLedgerLiveConnector = orangeKit.getOrangeKitLedgerLiveConnector({
+  ...connectorConfig,
+  options: {
+    tryConnectToAddress: getLastUsedBtcAddress(),
+  },
+})
+
+const embedConnectorsMap: Record<EmbedApp, () => CreateOrangeKitConnectorFn> = {
+  "ledger-live": orangeKitLedgerLiveConnector,
+}
 
-const connectors = [
+let createEmbedConnectorFn
+const embeddedApp = referralProgram.getEmbeddedApp()
+if (referralProgram.isEmbedApp(embeddedApp)) {
+  createEmbedConnectorFn = embedConnectorsMap[embeddedApp as EmbedApp]
+}
+
+const defaultConnectors = [
   orangeKitOKXConnector(),
   orangeKitUnisatConnector(),
   orangeKitXverseConnector(),
-] as unknown as CreateConnectorFn[]
+]
+
+const connectors = (createEmbedConnectorFn !== undefined
+  ? [createEmbedConnectorFn()]
+  : defaultConnectors) as unknown as CreateConnectorFn[]
```

Comment:
We need an update here - I can use the dapp w/o access code when passing any value via URL  eg. `http://localhost:5173/?embed=test`.